src/deepsparse/yolo/pipelines.py

```diff
@@ -197,9 +203,25 @@ def process_inputs(
             iou_thres=inputs.iou_thres,
             conf_thres=inputs.conf_thres,
             multi_label=inputs.multi_label,
+            original_image_shapes=original_image_shapes,
         )
         return [image_batch], postprocessing_kwargs
 
+    def _scale_boxes(
+        self, boxes: numpy.ndarray, original_image_shape: Optional[Tuple[int, ...]]
+    ):
+        if not original_image_shape:
+            return boxes
+
+        scale = numpy.flipud(
+            numpy.divide(
+                numpy.asarray(original_image_shape), numpy.asarray(self.image_size)
+            )
+        )
+        scale = numpy.concatenate([scale, scale])
```

Comment:
I think it may not be obvious for the arbitrary reader why we did this :)

Reply:
I'll add in a comment